db/migrate/20170926210603_add_episode_tags.rb

```diff
@@ -0,0 +1,16 @@
+class AddEpisodeTags < ActiveRecord::Migration
+  def up
+    add_column :episodes, :season_number, :integer
+    add_column :episodes, :episode_number, :integer
+    add_column :episodes, :itunes_type, :string, default: 'full'
+    execute "UPDATE episodes SET itunes_type = 'full'"
+    execute "UPDATE episodes SET itunes_type = 'bonus' WHERE title LIKE '%bonus%'"
+    execute "UPDATE episodes SET itunes_type = 'trailer' WHERE title LIKE '%trailer%'"
```

Comment:
I'm not sure this is a good idea, I think maybe they should have to mark these themselves.